src/functional/iterator/embedded.py

```diff
@@ -783,7 +827,7 @@ def impl(*iters: ItIterator):
     return impl
 
 
-def fendef_embedded(fun, *args, **kwargs):  # noqa: 536
+def fendef_embedded(fun, *args, **kwargs):
```

Comment:
Missing type annotations

Reply:
If you like to help me make it fancy, we can try to match `*args` of `fendef_embedded` to `*args` of fun...
For now I do `fun: Callable[..., None], *args: Any, **kwargs: Any`